### PR TITLE
Update as11fixity.py

### PR DESCRIPTION
**Pull request by AnjaMahler**
pseudo update and progress

### CODE REVIEW

**Comment by kieranjol** [comment]
Hi Anja,

Thanks for sending this. I'm finding the comments  a little difficult to follow as I keep having to scroll down to match the numbers to the pseudocode, so instead of:

```
 starting_dir = sys.argv[1]
 #2

```

it could be:

```
#2 Define path to starting folder
starting_dir = sys.argv[1]
```

It's also easier to read if the #comment is above the ode that it's describing, rather than below. You should be able to edit the code in your pull request on the github site itself, give us a shout if you need a hand.

**Comment by AnjaMahler** [comment]
Thanks :). Will amend and get back you.

**Comment by kieranjol** [comment]
Awesome, and I should have acknowledged earlier that it was my idea to
comment all the code with the numbers, so I apologise for wasting your time
earlier!! My blood sugar was low or something ¯\_(ツ)_/¯

**Comment by AnjaMahler** [comment]
No no, my headache's fold or spider's spirit revenge ^^

**Comment by kieranjol** [comment]
that poor spider... Never knew you had it in you Anja, though obviously
Verena is to blame.

**Comment by ecodonohoe** [comment]
http://giphy.com/gifs/fire-with-kill-wvFrJa4aWKuvm

**Comment by AnjaMahler** [comment]
http://giphy.com/gifs/home-alone-tarantula-burglar-apw8IpnNDwUHC

**Comment by kieranjol** [comment]
😆 💪 👣 👠

**Comment by kieranjol** [comment]
HI Anja, looks like this can't be merged right now, possibly because it's an old fork, or there's some conflict with Eoin's recent Pull Request. We should find a way to add this tomorrow.

File: as11fixity.py [hunk 1]
```diff
@@ -33,43 +39,96 @@ def create_csv(csv_file, *args):
...
+     
+        
+        
+def md5(full_path):
+    hash_md5 = hashlib.md5()
+    with open(full_path, "rb") as f:
+        for chunk in iter(lambda: f.read(4096), b""):
+            hash_md5.update(chunk)
+    return hash_md5.hexdigest()
+print md5(full_path)
```
Comment (kieranjol) [inline as11fixity.py]: This is running the checksum and printing it to the screen

File: as11fixity.py [hunk 2]
```diff
@@ -33,43 +39,96 @@ def create_csv(csv_file, *args):
...
+#13
+
+if md5(full_path)==checksum:
```
Comment (kieranjol) [inline as11fixity.py]: this is running it again. Instead, if you store the mxf md5 as a variable once, then you can runn the comparison on it after.
SOmething like
mxf_md5 = md5(full_path)
```
if mxf_md5 == checksum:
    print 'ALL IS WELL'
else:
    print 'OH JESUS'
```

**Comment by kieranjol** [comment]
Hey, i'm closing this as it's now out of date

**Pull request closed without merging**